dask/bytes/tests/test_local.py

```diff
@@ -70,7 +70,7 @@ def test_urlpath_inference_errors():
         get_fs_token_paths([])
 
     # Protocols differ
-    with pytest.raises(ValueError, match="the same protocol"):
+    with pytest.raises(ValueError, match="Protocol mismatch"):
```

Comment:
This seems to be testing a purely fsspec function. @martindurant is there coverage for this in fsspec itself? If so, I think we just remove this here.

Reply:
I'll put in this exact case.

Reply:
We have a few other tests in there that are directly testing `get_fs_token_paths` -- should these all be moved to `fsspec`?

Reply:
When originally this code was factored out into fsspec, we thought it would be good to keep testing them here, although many were copied or adapted. I'm not sure if it's the case anymore.

Reply:
I'm going to go ahead and strip out tests that are duplicated upstream.